daffodil-tdml/src/test/scala/org/apache/daffodil/tdml/UnitTestTDMLRunner.scala

```diff
@@ -0,0 +1,441 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
FYI: These are not new tests. They came from TestTDMLRunner.scala which is now in the tests of daffodil-tdml-processor. 

This file is the unit tests for the daffodil-tdml, which does not include a processor. So these are true unit tests. They can't run an actual test. They are unit tests for the piece parts of TDML files, and the various things that get computed about a test case, without actually running a TDML test. 

These tests used to be in the same file with a bunch of tests that DO actually run the TDML runner. They were split out so that these can run without the need of a TDML processor.